go/vt/vtgate/planbuilder/operators/table.go

```diff
@@ -89,7 +88,7 @@ func (to *Table) TablesUsed() []string {
 func addColumn(op ColNameColumns, e sqlparser.Expr) (int, error) {
 	col, ok := e.(*sqlparser.ColName)
 	if !ok {
-		return 0, vterrors.Errorf(vtrpcpb.Code_INTERNAL, "can't push this expression to a table/vindex")
+		return 0, vterrors.VT13001("can't push this expression to a table/vindex")
```

Comment:
cannot

Reply:
resolved in [f2351bb](https://github.com/vitessio/vitess/pull/10738/commits/f2351bba65f347a8e5ce72ff87757ccaf20115af)